modules/op-resolver-cluster-tasks-ref.adoc

```diff
@@ -257,6 +257,68 @@ spec:
 * All result names were changed to uppercase.
 * The `gitInitImage` parameter was removed.
 
+[discrete]
+[id="op-taskref-jib-maven_{context}"]
+== jib-maven
+
+The `jib-maven` task builds Java, Kotlin, Groovy, and Scala source into a container image using the Jib tool that works with Maven projects.
```

Comment:
I took this line from the yaml file, but I am not sure about the support downstream, @aneeshmbhat could you please let me know?? Thank you!